quickstarts/snyk/config.yml

```diff
@@ -0,0 +1,49 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: snyk
+
+# Displayed in the UI (required)
+title: Snyk
+
+# Long-form description of the quickstart (required)
+description: |
+  The Snyk quickstart allows you to send application security vulnerabilities of your services into New Relic.
```

Comment:
It might help to add a bit more description here. Like how does this integration work (this one looks specific to Azure so it should probably be mentioned).